blocks-common/i-bem/__dom/_init/i-bem__dom_init_auto.deps.js

```diff
@@ -0,0 +1,6 @@
+({
+    shouldDeps : [
+        'next-tick',
```

Comment:
This dependency will add another 100 lines of js. Even on the projects that are not using `ym`.

Could we replace usage of `next-tick` block with `BEM.afterCurrentEvent()`.